source/libs/page-detect.ts

```diff
@@ -374,7 +374,7 @@ export const _isBranches = [
 export const isUserProfile = (): boolean => select.exists('.user-profile-nav');
 export const _isUserProfile = domBased;
 
-export const isUserProfileRepoTab = (): boolean => location.search.startsWith('?tab=repositories');
+export const isUserProfileRepoTab = (): boolean => location.search.startsWith('?tab=repositories') && location.pathname.length > 1;
```

Comment:
?

This needs to include isUserProfile.

Reply:
Working on it